app/angular/src/client/preview/angular-beta/AbstractRenderer.ts

```diff
@@ -92,24 +92,27 @@ export abstract class AbstractRenderer {
    * @param forced {boolean} If :
    * - true render will only use the StoryFn `props' in storyProps observable that will update sotry's component/template properties. Improves performance without reloading the whole module&component if props changes
    * - false fully recharges or initializes angular module & component
+   * @param component {Component}
    * @param parameters {Parameters}
    */
   public async render({
     storyFnAngular,
     forced,
     parameters,
+    component,
     targetDOMNode,
   }: {
     storyFnAngular: StoryFnAngularReturnType;
     forced: boolean;
+    component?: any;
```

Comment:
I would tend to say that the correct ts type for the reference to an angular component class is : 
Type<unknown>

I'm not sure of myself. but it can avoid `any` and show that we are referring to an angular component
🙃